kustomize/dev/ethereum/charts/ethereum/templates/configmap-node.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: v1
```

Comment:
Just calling out that you have separate resources (service and sts) for the consensus / execution concepts. But they share a single configmap.